packages/app/src/debug/DebugContainer.cy.tsx

```diff
@@ -70,6 +122,7 @@ describe('<DebugContainer />', () => {
         render: (gqlVal) => {
           return (
             <DebugContainer
+              loading={false}
```

Comment:
I ended up removing this prop anyway because I moved the loading state outside to `Debug.vue` instead of in the container. This makes the transitioning easier